go/teams/member_set.go

```diff
@@ -31,42 +31,42 @@ func newMemberSet(ctx context.Context, g *libkb.GlobalContext, req keybase1.Team
 
 func (m *memberSet) loadMembers(ctx context.Context, g *libkb.GlobalContext, req keybase1.TeamChangeReq) error {
 	var err error
-	m.Owners, err = m.loadGroup(ctx, g, req.Owners)
+	m.Owners, err = m.loadGroup(ctx, g, req.Owners, true)
```

Comment:
I'm just using that type to shuttle team membership from the API server to Gregor.